configuration/enable-powerBI-data-collection.md

```diff
@@ -0,0 +1,77 @@
+---
+description: This article explains how to enable Power BI data collection for SysKit Point.
+---
+
+# Enable Power BI Data Collection
+
+Enabling Power BI data collection for SysKit Point requires initial setup through your Microsoft 365 Azure Portal. 
+
+## Create a Power BI Security Group 
+
+ * **Click the [Azure Active Directory (1)](https://portal.azure.com/#view/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/~/Overview)**; on the Home page of your Azure portal. 
+ * **Select Groups (2)**; located under the Manage section on the left side of the screen. Select Groups
+ * **Click the New group button (3)** on the top bar. 
+ 
+![Azure Active Directory - Create New Group](../.gitbook/assets/enable-powerbi-data-collection_aad-newgroup.png)
+
+Fill out the following details when creating the new group:
+
+ * Under **Group Type (1)**, select **Security (2)**
+ * **Name (3)** the group **Power BI Group** or similar
+ * For the **Group Description (4)** list the purpose of the new group
```

Comment:
dodati još na kraj retka, for example 'SysKit Point Power BI data collection group' ili tome slično